pkg/operator/ceph/cluster/cluster_external.go

```diff
@@ -112,6 +112,15 @@ func (c *ClusterController) configureExternalCephCluster(cluster *cluster) error
 		return errors.Wrap(err, "failed to create csi config map")
 	}
 
+	// update the msgr flag
+	if len(cluster.ClusterInfo.Monitors) > 0 {
```

Comment:
You don't really need this check for length. The `for` loop will be a noop if there are no mons.